packages/fx-core/src/plugins/resource/aad/aadAppClient.ts

```diff
@@ -228,6 +229,29 @@ export class AadAppClient {
     }
   }
 
+  public static async grantPermission(
+    ctx: PluginContext,
+    stage: string,
+    objectId: string,
+    userObjectId: string
+  ): Promise<void> {
+    try {
+      await GraphClient.grantPermission(TokenProvider.token as string, objectId, userObjectId);
+    } catch (error) {
+      if (error?.response?.data?.error.message == Constants.createOwnerDuplicatedMessage) {
+        return;
```

Comment:
We can add logs here to inform user that the owner has already been added

Reply:
Fixed. Thx!